modules/doc/content/getting_started/examples_and_tutorials/tutorial01_app_development/step07_test_harness.md

```diff
@@ -0,0 +1,12 @@
+# Step 7: Develop an Application Test
+
+!alert construction
+The remainder of this tutorial is currently being developed. More content should be available soon. For now, refer back to the [examples_and_tutorials/index.md] page for other helpful training materials or check out the MOOSE [application_development/index.md] pages for more information.
+
+<!--use the same mesh as simple diffusion-->
+
+<!--turn pressure_diffusion.i into a test. Explain how to run in parallel and demonstrate. Introduce postprocessors to output results and explain how postprocs will be the subject of a subsequent step. We already know this solution to be good and true, so its a good test of our DarcyPressyre object.-->
```

Comment:
Can you make running in parallel a stand-alone step: Step 7 
Then make Step 8: The testing step. First, explain the testing then put it in practice by adding the permeability error check (see above) and add a RunException test

Reply:
Maybe I just won't use `addRangeCheckedParam()` altogether, just `addParam()`, then add the manual check in Step 8